tools/check_release.py

```diff
@@ -376,15 +419,19 @@ def get_latest_library_versions(repo_path):
     logging.basicConfig(level=level)
     
     # Read configuration data
-    json_data = json.load(open(os.path.join(os.path.dirname(__file__), "check_release.json")))
-    
+    json_data = json.load(open(os.path.join(os.path.dirname(__file__), 
+                               "check_release.json")))
```

Comment:
We should probably close the file after opening it for reading. 

```
json_file = open(os.path.join(os.path.dirname(__file__), "check_release.json"))
json_data = json.load(json_file)
json_file.close()
```

Or you can always do:

```
json_data  = None
with open(os.path.join(os.path.dirname(__file__), "check_release.json")) as data:
     json_data = data
```

or something similar